doc/sphinx/tutorial/getting-started-with-mir.md

```diff
@@ -0,0 +1,129 @@
+# Getting started with Mir
+
+This tutorial will guide you through Mir's basic functionality. 
+
+By the end of the tutorial, you will install and run a demo application, learn how to use Mir in different environments, and learn about features that Mir provides for Mir-based compositors. 
+
+If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing compositors.
+
+
+## Installing demo applications
+Mir demos are available on Debian derivatives, Fedora, and Alpine. For distros
+that don't have pre-built binaries, examples can be built from source.
+
+### On Debian and its derivatives
+
+```sh
+sudo apt install mir-demos mir-graphics-drivers-desktop
+```
+
+### On Fedora
+```sh
+sudo dnf install mir-demos
+```
+
+### On Alpine
+```sh
+sudo apk add mir-demos mir
+```
+
+## Running applications on X11 or Wayland
+
+We'll work with `miral-app`, a script that handles running a Mir shell with a basic GUI.
+
+Mir allows you to run a graphical shell in *desktop mode* or in *kiosk mode*.
```

Comment:
```suggestion
`miral-app` allows you to run a graphical shell in *desktop mode* or in *kiosk mode*, among other modes. Mir also gives you the ability to write your own mode.
```

Reply:
stet. (This tutorial is about running the examples, not writing bespoke compositors.)